src/vm/i386/excepcpu.h

```diff
@@ -90,15 +91,24 @@ extern VOID ResetSEHRecord(PEXCEPTION_REGISTRATION_RECORD record);
 
 #endif
 
+
+PEXCEPTION_REGISTRATION_RECORD GetCurrentSEHRecord();
```

Comment:
``GetCurrentSEHRecord`` and ``GetFirstCOMPlusSEHRecord``is not a part of #8613 (It was there from the beginning).
#8613 introduces only ``SetSEHRecord`` and ``ResetSEHRecord``.